ckcp/gitops.sh

```diff
@@ -245,7 +242,7 @@ secrets:
     rm "$argocd_yaml"
     # </workaround>
 
-    sed -i -e 's:admin$:admin_kcp:g' "$KUBECONFIG_KCP"
+    sed -i'' -e 's:admin$:admin_kcp:g' "$KUBECONFIG_KCP"
```

Comment:
should be both [Linux and Mac compatible](https://stackoverflow.com/a/4247319/3894898)